### DISCUSSION TITLE
KNX Support for VdMot Controller

### DISCUSSION THREAD

**Discussion by Lenti84** [Lenti84/VdMot_Controller#108] [Polls]
There we're some questions about KNX Support of VdMot Controller.
With this poll I want to estimate the demand for that feature before I maybe start implementing.

Feel free to contribute with ideas and even better with some work on the code, documentation or testing.

Kind regards
Andreas

**Comment by shivan** [comment]
@Lenti84 are you talking about KNX via cable or KNX IP?

For me KNX IP is not needed, then I can also use WiFi. But since I currently have a thermo electric floor heating controlled via KNX cable that would be of interest for me. But I don't know how easy that is or how it could affect the current KNX network.

**Reply by Lenti84** [reply]
KNX via cable.

Implementation can bei made via already existing KNX Bus couplers from e.g. Siemens. Hopefully they can be connected directly to the extension interface of the VdMot Controller.
On software side the are some libraries supporting this way of KNX integration into own projects.
